packages/server/src/course/course.entity.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
entities need this, since they have the `(type) => asdfasdfa` decorator on relations